lib/wick-get-iface-ip

```diff
@@ -33,7 +33,17 @@ wickGetIfaceIp() {
         allInterfaces=true
     fi
 
-    if wickCommandExists ifconfig; then
+    # Works for systemd distros
+    if wickCommandExists networkctl; then
+        cmd=(networkctl status)
+
+	if [[ -n "$interface" ]]; then
+	    cmd[${#cmd[@]}]="$interface"
+	fi
+
+       # FIXME: Unfortunately, IPv4 only
```

Comment:
Submitted code should not have bugs, thus can you fix this before committing it to the repository?